command.go

```diff
@@ -626,13 +623,8 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 		return c.Root().ExecuteC()
 	}
 
-	if EnableWindowsMouseTrap && runtime.GOOS == "windows" {
-		if mousetrap.StartedByExplorer() {
-			c.Print(MousetrapHelpText)
-			time.Sleep(5 * time.Second)
-			os.Exit(1)
-		}
-	}
+	// windows hook
+	preExecHook(c)
```

Comment:
It's bikeshedding, but we already have user exposed things called PreExec stuff. I'm trying to think of a better name, but I don't love this one.